ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -1267,3 +1267,12 @@ private final class DeallocationToken {
 		observer.sendCompleted()
 	}
 }
+
+extension SignalProducerType where Value: Equatable {
```

Comment:
I think it should not be necessary to constraint `Value` to conform to `Equatable`, or is there?